mypy/config_parser.py

```diff
@@ -439,6 +439,19 @@ def parse_section(
         options_key = key
         if key in config_types:
             ct = config_types[key]
+        elif key in ("enabled_error_codes", "disabled_error_codes"):
+            # These should be "enable_error_code" and "disable_error_code". But because these fields
+            # exist on Options, we would otherwise accept them here and crash later.
+            suggestion = {
+                "enabled_error_codes": "enable_error_code",
+                "disabled_error_codes": "disable_error_code",
+            }
```

Comment:
Could we move the dict outside of the loop and check against e.g. `key in option_replacements` to avoid the repetition?